modules/ob-installing-builds-by-using-the-web-console.adoc

```diff
@@ -29,25 +29,24 @@ You can install the {builds-operator} from the OperatorHub by using the {builds-
 
 ** The *Update Channel* is set to *latest* by default. The latest channel enables installation of the most recent stable version of the {builds-operator}.
 ** To install a specific version of the {builds-operator}, cluster administrators can use the corresponding `builds-<version>` channel. For example, to install the {builds-operator} version `1.3`, you can use the `builds-1.3` channel.
+
 . Click *Install*.
 
 .Verification
 
-After installing the Builds for Red Hat OpenShift Operator, you must verify that the required resources are created to ensure proper functioning of the controllers.
+After installing the {builds-title}, perform the following steps to verify that the required resources are created to ensure proper functioning of the controllers.
 
-.Procedure
 . In the *Administrator* perspective of the web console, select the project as openshift-builds.
 
-. Navigate to the *Operators* -> *Installed Operators* page.
+. Navigate to the *Operators* -> *Installed Operators* page. 
 
-. Click the *All instances* tab. 
+. On the *Installed Operators* page, click the *All instances* tab. 
 
-. On the *All instances* tab, verify that the `OpenShiftBuild` and `ShipwrightBuild` instances are listed.
+. On the *All instances* tab, verify that the *Status* of `OpenShiftBuild` and `ShipwrightBuild` operator is displayed as *Succeeded Up to date*.
```

Comment:
Change `ShipwrightBuild` operator to `ShipwrightBuild` Operator. Also, I do not think we need to bold the *Status* as it is simply signifying the status. Please confirm.